src/transformStateWithClones.js

```diff
@@ -64,7 +64,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const newState = {...state};
+  let statesHistory  = [];
+  for (let i = 0; i < transforms.length; i++) {
+    const { properties } = transforms[i];
+
+    switch (transforms[i].operation) {
+      case 'addProperties': {
+
+        for (const key in properties) {
+          newState[key] = properties[key];
+        }
```

Comment:
пустий рядок після {}
```suggestion
        }
        
```